plugins/callback/opentelemetry.py

```diff
@@ -0,0 +1,374 @@
+# (C) 2021, Victor Martinez <VictorMartinezRubio@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    author: Victor Martinez (@v1v)  <VictorMartinezRubio@gmail.com>
+    name: opentelemetry
+    type: notification
+    short_description: create distributed traces with OpenTelemetry.
+    version_added: historical
+    description:
+      - This callback create distributed traces for each ansible task with OpenTelemetry.
+    options:
+      include_setup_tasks:
+        name: Include setup tasks
+        default: True
+        description: Should the setup tasks be included in the distributed traces
```

Comment:
```suggestion
        description:
          - Should the setup tasks be included in the distributed traces.
```
Better use lists than strings, and make sure that every entry is a proper sentence / paragraph.